Mathlib/AlgebraicGeometry/EllipticCurve/Weierstrass.lean

```diff
@@ -181,6 +181,54 @@ lemma c_relation : 1728 * W.Δ = W.c₄ ^ 3 - W.c₆ ^ 2 := by
 
 end Quantity
 
+section ModelsWithJInvariant_0_1728
+
+variable (R : Type u) [CommRing R]
```

Comment:
You don't need to reinstantiate everything if you want to make `R` explicit - you can just do `variable (R)`. Notice that in every goal now we have two `CommRing R`'s.